modules/bomb_word.py

```diff
@@ -67,23 +127,36 @@ def bomb_word(bot, update, args, job_queue, chat_data):
     word = normalize_text(str(args[0]))[0]
     user_id = update.message.from_user.id
 
-    if 'bombs' not in chat_data:
-        chat_data['bombs'] = {}
-
-    if 'bomb_pidors' not in chat_data:
-        chat_data['bomb_pidors'] = {}
+    initialize_containers(chat_data)
 
     log.debug(chat_data['bombs'])
     if user_id in chat_data['bombs']:
         update.message.reply_text('Ска не еби до завтра!')
     elif len(word) < MIN_LENGTH:
         update.message.reply_text('Слово слишком короткое, как и твой член!')
     else:
-        chat_data['bombs'][user_id] = word
+        chat_data['bombs'][user_id] = {}
+        chat_data['bombs'][user_id]['word'] = word
```

Comment:
i think it is better to create Bomb class to represent bomb instead of dictionary, because it becomes too complicated now

Reply:
feel free to create a separate issue for it.